vernac/auto_ind_decl.ml

```diff
@@ -458,7 +465,7 @@ let do_replace_lb aavoid narg p q =
     let sigma = Tacmach.New.project gl in
     let env = Tacmach.New.pf_env gl in
     let u,v = destruct_ind env sigma type_of_pq in
-    find_scheme (!lb_scheme_kind_aux ()) (fst u) (*FIXME*) >>= fun c ->
+    get_scheme (!lb_scheme_kind_aux ()) (fst u) >>= fun c ->
```

Comment:
Do we know what the fixme was about?

Reply:
I believe this is a leftover of a dynamic test that led to #9838. That is, this call was generating a scheme dynamically when it should have not.